Examples/HelloWorld/CMakeLists.txt

```diff
@@ -0,0 +1,8 @@
+
```

Comment:
Add all of the examples as tests, similar to the SimpleGaussian example [CMakeLists file](https://github.com/SimpleITK/SimpleITK/blob/next/Examples/SimpleGaussian/CMakeLists.txt).